item-detail.html

```diff
@@ -421,8 +421,134 @@
       </div>
     </div>
 
+    <!-- 埋め込みJSでは、兄弟要素を書いたときにエラー時の処理で兄弟が消えるケースがあるので、div でくくります。 Diffが見やすいようにインデントしていません-->
+    <!-- cms-content-typeには実際のコンテンツタイプのIDを指定します-->
+    <div>
+    <div class="p-product-body" cms-item cms-content-type="color-me-items" style="visibility: hidden;">
```

Comment:
```suggestion
    <div class="p-product-body" cms-item cms-content-type="sample-ab-testing" style="visibility: hidden;">
```

サンプルで自動生成されたコンテンツタイプをそのまま利用いただいていると思うので、上記の変更が必要そうです！
![スクリーンショット 2023-09-26 14 47 21](https://github.com/ecogrammer/shinzaburo-colorme-template/assets/11781291/422d8ee2-1d82-4552-8c21-c6de488890f7)